scripts/delete-account.js in fxa-auth-server is broken again

I haven't been following the Typescript conversions on fxa-auth-server, but I think we might have broken `scripts/delete-account.js` again. ([Previously](https://github.com/mozilla/fxa/issues/4032)) A few other scripts affected too, like `clean-up-old-ci-stripe-customers.js`

Looks like `config/index.js` is now `config/index.ts`, which leads to this:
```
➜  fxa-auth-server git:(master) ✗ node scripts/delete-account.js lorchard@mozilla.com
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module '../config'
Require stack:
- /home/lmorchard/devel/fxa/packages/fxa-auth-server/scripts/delete-account.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/lmorchard/devel/fxa/packages/fxa-auth-server/scripts/delete-account.js:25:16)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/lmorchard/devel/fxa/packages/fxa-auth-server/scripts/delete-account.js'
  ]
}
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1857)
┆Issue Number: FXA-1857

--- a/packages/fxa-auth-server/scripts/activate-new-signing-key.js
+++ b/packages/fxa-auth-server/scripts/activate-new-signing-key.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js
+++ b/packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js
@@ -1,15 +1,15 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const program = require('commander');
-const package = require('../package.json');
+const pckg = require('../package.json');
 
 async function init() {
   program
-    .version(package.version)
+    .version(pckg.version)
     .option('-c, --config [config]', 'Configuration to use. Ex. dev')
     .option(
       '-k, --stripe-key [key]',

--- a/packages/fxa-auth-server/scripts/delete-account.js
+++ b/packages/fxa-auth-server/scripts/delete-account.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
@@ -8,7 +8,7 @@
 
  Usage:
 
- node scripts/delete-account.js user@example.com [...]
+ scripts/delete-account.js user@example.com [...]
 
  This script is used to manually delete a user's account,
  e.g. in response to a GDPR deletion request.  It uses the

--- a/packages/fxa-auth-server/scripts/find-stripe-sync-issues.js
+++ b/packages/fxa-auth-server/scripts/find-stripe-sync-issues.js
@@ -1,11 +1,11 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const program = require('commander');
-const package = require('../package.json');
+const pckg = require('../package.json');
 const config = require('../config').getProperties();
 const StatsD = require('hot-shots');
 
@@ -24,7 +24,7 @@ function isNotFoundError(err) {
 
 async function init() {
   program
-    .version(package.version)
+    .version(pckg.version)
     .option('-c, --config [config]', 'Configuration to use. Ex. dev')
     .option(
       '-k, --stripe-key [key]',

--- a/packages/fxa-auth-server/scripts/gen_keys.js
+++ b/packages/fxa-auth-server/scripts/gen_keys.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/gen_vapid_keys.js
+++ b/packages/fxa-auth-server/scripts/gen_vapid_keys.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/must-reset.js
+++ b/packages/fxa-auth-server/scripts/must-reset.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
@@ -8,7 +8,7 @@
 
  Usage:
 
- node scripts/must-reset.js -i ./reset.json
+ scripts/must-reset.js -i ./reset.json
 
  This script is used to put user accounts into a "must reset" state. It uses the
  same config file as key_server.js so should be run from a production instance.

--- a/packages/fxa-auth-server/scripts/oauth_gen_keys.js
+++ b/packages/fxa-auth-server/scripts/oauth_gen_keys.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/populate-stripe-customer-cache.js
+++ b/packages/fxa-auth-server/scripts/populate-stripe-customer-cache.js
@@ -1,15 +1,15 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const program = require('commander');
-const package = require('../package.json');
+const pckg = require('../package.json');
 
 async function init() {
   program
-    .version(package.version)
+    .version(pckg.version)
     .option('-c, --config [config]', 'Configuration to use. Ex. dev')
     .option(
       '-k, --stripe-key [key]',

--- a/packages/fxa-auth-server/scripts/prepare-new-signing-key.js
+++ b/packages/fxa-auth-server/scripts/prepare-new-signing-key.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/retire-old-signing-key.js
+++ b/packages/fxa-auth-server/scripts/retire-old-signing-key.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/sms/send.js
+++ b/packages/fxa-auth-server/scripts/sms/send.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/test-remote-quick.js
+++ b/packages/fxa-auth-server/scripts/test-remote-quick.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node-script
 
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this

--- a/packages/fxa-auth-server/scripts/verification-reminders.js
+++ b/packages/fxa-auth-server/scripts/verification-reminders.js
@@ -1,3 +1,5 @@
+#!/usr/bin/env ts-node-script
+
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */

--- a/packages/fxa-auth-server/scripts/write-emails-to-disk.js
+++ b/packages/fxa-auth-server/scripts/write-emails-to-disk.js
@@ -1,3 +1,5 @@
+#!/usr/bin/env ts-node-script
+
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
@@ -7,7 +9,7 @@
  * and ./.mail_output/<email type>.txt
  *
  * Usage:
- * node ./scripts/write-to-disk.js
+ * ./scripts/write-to-disk.js
  *
  * Emails that are written to disk can be previewed in Firefox
  * to give a rough idea of how they would render in real life.

